addons/mail/static/src/core/common/message.js

```diff
@@ -134,16 +134,16 @@ export class Message extends Component {
         );
         useEffect(
             () => {
-                if (!this.shadowRoot) {
+                if (!this.shadowRoot || !this.message.type || this.message.type.includes("email")) {
```

Comment:
I'm not sure if I got it wrong, but shouldn't it be negative condition on email here?

Reply:
Oops, made the mistake, fixing it (the screenshot were taken before this typo, hence the correct visual)